CMakeLists.txt

```diff
@@ -163,7 +163,7 @@ function(RELATIVE_PROTOBUF_GENERATE_CPP NAME SRCS HDRS ROOT_DIR DEPEND)
     set(ABS_FILE ${ROOT_DIR}/${INFILE})
     get_filename_component(FILE_DIR ${ABS_FILE} DIRECTORY)
     get_filename_component(FILE_WE ${INFILE} NAME_WE)
-    if(ONNX_ML)
+    if(ONNX_ML AND NOT(FILE_WE STREQUAL "onnx-data"))
```

Comment:
Add comment here to explain why you are adding this exception for onnx-data

Reply:
Addressed